aten/src/ATen/native/README.md

```diff
@@ -384,6 +390,72 @@ will be automatically differentiated! This can be the case if the function imple
 only calls other operations which are themselves differentiable.  In this
 case, you don't have to write an entry in `tools/autograd/derivatives.yaml`.
 
+### Choosing the right dispatch keyword
+
+After writing a native function in C++, it's important to think about which dispatch keyword
+to use in native_functions.yaml as it gives dispatcher information about backend and autograd support
+of the implementation.
+
+Here're steps to follow to decide the right dispatch keyword:
+
+1. Think about inference, does your kernel work for all backends?
```

Comment:
colon instead of comma; comma here is a comma splice because the two sides are complete sentences